plutus-contract/src/Plutus/Contract.hs

```diff
@@ -97,6 +97,8 @@ module Plutus.Contract(
     , Request.isTxConfirmed
     -- ** Tx output confirmation
     , Request.awaitTxOutStatusChange
+    -- * Parameters
+    , Request.getParams
```

Comment:
Oh, and I would suggest the name `getNodeParams` just for clearness

Reply:
Hmm shouldn't `Params` be renamed `NodeParams` then too? (I actually never thought of `Params` as just node parameters, but that's indeed what they are.)

Reply:
I can do that in the PR that fixes the PAB to actually query the node, until then `getNodeParams` could falsely imply you'd get the actual values from the node.

Reply:
Oh, I though we had `NodeParams`, that's why I proposed it :D Then good point about chaning `Params` to `NodeParams`